test/suites/basic.sh

```diff
@@ -672,6 +672,27 @@ test_basic_usage() {
   lxc profile delete foo
   lxc delete -f c1
 
+  # Test assigning a profile through a YAML file to an instance.
+  poolName=$(lxc profile device get default root pool)
+  lxc profile create foo < <(cat <<EOF
+config:
+  limits.cpu: 2
+  limits.memory: 1024MB
+description: Test profile
+devices:
+  root:
+    path: /
+    pool: ${poolName}
+    type: disk
+EOF
+)
+  lxc init testimage c1
```

Comment:
you can use `--profile` rather than needing to use `lxc profile assign`.

Reply:
fixed